src/envoy/http/service_control/filter_stats.h

```diff
@@ -37,6 +37,7 @@ namespace service_control {
   COUNTER(denied_consumer_error)         \
   COUNTER(denied_consumer_quota)         \
   COUNTER(denied_producer_error)         \
+  COUNTER(denied_by_no_route)            \
```

Comment:
why do we need to add this stat here?  I don't see where it is used

Reply:
Done.